terraform/aws-public/mesos-slaves.tf

```diff
@@ -8,7 +8,7 @@ resource "atlas_artifact" "mesos-slave" {
 /* Mesos slave instances */
```

Comment:
could we not do some kind of map here in the variables?

``
amis = {
 eu-west-1 => 'ami-123',
 us-east-1 => 'ami-456',
}
```
etc..

and then use that var to replace on the end..

```
 "${atlas_artifact.mesos-master.metadata_full.region-${var.amis.${region}}}"
```
or whatever is is